spec/faraday/adapter/net_http_persistent_spec.rb

```diff
@@ -40,4 +40,26 @@
     # `pool` is only present in net_http_persistent >= 3.0
     expect(http.pool.size).to eq(5) if http.respond_to?(:pool)
   end
+
+  context 'min_version' do
+    let(:conn_options) do
+      {
+        headers: { 'X-Faraday-Adapter' => adapter },
+        ssl: {
+          min_version: '1.0'
```

Comment:
IMO it's nice that tests can serve as documentation, a value like `:TLS1_2` (which is [mapped](https://github.com/ruby/openssl/blob/01b23fa8eee2ff9cc4ef5f6fabca6d999e0979c2/ext/openssl/ossl_ssl.c#L120-L127) to `OpenSSL::SSL::TLS1_2_VERSION`) would be a real-life example that can be used here.

Reply:
Thanks for chiming in! Alright, let's use a realistic value here, it wont hurt readability, or fool anyone.